projects/Rosenpass/services/basic/examples/basic.nix

```diff
@@ -0,0 +1,5 @@
+{ ... }:
+
+{
+  services._serviceName_.enable = true;
```

Comment:
It's not enough to copy the template, you need to modify it as well... `_serviceName_` should be replace with the actual service  name